src/main/java/gg/solarmc/kitpvp/kill/damage/DamageHolder.java

```diff
@@ -1,10 +1,10 @@
-package gg.solar.kitpvp;
+package gg.solarmc.kitpvp.kill.damage;
 
+import gg.solarmc.kitpvp.KitpvpPlugin;
```

Comment:
This class needs some work. I see a few areas for improvement.

First, a Player should never be retained in a Map beyond the time when the player quits. To solve this you can key by UUID instead.

Second, you can replace the containsKey/get + cancel/put calls with a single `compute` call. The performance is not relevant; accessing the map once is about code quality.

Reply:
This class was a mockup for the logic. I'll switch it to user ids' eventually but the logic inside is currently not final

Reply:
Keying by weak references is cringe anyways the map should be emptied automatically regardless thanks to the listener